public/bitemp_graph_helper.js

```diff
@@ -147,12 +161,34 @@ var removeButtonEvents = function () {
   $('#viewButton').unbind('click');
   $('#saveButton').unbind('click');
   $('#change-prop').unbind('click');
+};
+
+var drawChart = function (params, docProp) {
+  var chart = barChart()
+    .data(params.data)
+    .width(params.width)
+    .height(params.height)
+   // .valStart(params.valStart)
```

Comment:
It's eventually going to be added with Ashley and Hilary's code, they just don't have it functioning yet.